Extension/src/LanguageServer/settings.ts

```diff
@@ -25,139 +25,140 @@ nls.config({ messageFormat: nls.MessageFormat.bundle, bundleFormat: nls.BundleFo
+    vcFormatIndentCaseContents: boolean;
+    vcFormatIndentCaseContentsWhenBlock: boolean;
+    vcFormatIndentLambdaBracesWhenParameter: boolean;
+    vcFormatIndentGotoLabels: string;
+    vcFormatIndentPreprocessor: string;
+    vcFormatIndentAccesSpecifiers: boolean;
+    vcFormatIndentNamespaceContents: boolean;
+    vcFormatIndentPreserveComments: boolean;
+    vcFormatNewLineScopeBracesOnSeparateLines: boolean;
+    vcFormatNewLineBeforeOpenBraceNamespace: string;
+    vcFormatNewLineBeforeOpenBraceType: string;
+    vcFormatNewLineBeforeOpenBraceFunction: string;
+    vcFormatNewLineBeforeOpenBraceBlock: string;
+    vcFormatNewLineBeforeOpenBraceLambda: string;
+    vcFormatNewLineBeforeCatch: boolean;
+    vcFormatNewLineBeforeElse: boolean;
+    vcFormatNewLineBeforeWhileInDoWhile: boolean;
+    vcFormatNewLineCloseBraceSameLineEmptyType: boolean;
+    vcFormatNewLineCloseBraceSameLineEmptyFunction: boolean;
+    vcFormatSpaceWithinParameterListParentheses: boolean;
+    vcFormatSpaceBetweenEmptyParameterListParentheses: boolean;
+    vcFormatSpaceAfterKeywordsInControlFlowStatements: boolean;
+    vcFormatSpaceWithinControlFlowStatementParentheses: boolean;
+    vcFormatSpaceBeforeLambdaOpenParenthesis: boolean;
+    vcFormatSpaceWithinCastParentheses: boolean;
+    vcFormatSpaceAfterCastCloseParenthesis: boolean;
+    vcFormatSpaceWithinExpressionParentheses: boolean;
+    vcFormatSpaceBeforeBlockOpenBrace: boolean;
+    vcFormatSpaceBetweenEmptyBraces: boolean;
+    vcFormatSpaceBeforeInitializerListOpenBrace: boolean;
+    vcFormatSpaceWithinInitializerListBraces: boolean;
+    vcFormatSpacePreserveInInitializerList: boolean;
+    vcFormatSpaceBeforeOpenSquareBracket: boolean;
+    vcFormatSpaceWithinSquareBrackets: boolean;
+    vcFormatSpaceBeforeEmptySquareBrackets: boolean;
+    vcFormatSpaceBetweenEmptySquareBrackets: boolean;
+    vcFormatSpaceGroupSquareBrackets: boolean;
+    vcFormatSpaceWithinLambdaBrackets: boolean;
+    vcFormatSpaceBetweenEmptyLambdaBrackets: boolean;
+    vcFormatSpaceBeforeComma: boolean;
+    vcFormatSpaceAfterComma: boolean;
+    vcFormatSpaceRemoveAroundMemberOperators: boolean;
+    vcFormatSpaceBeforeInheritanceColon: boolean;
+    vcFormatSpaceBeforeConstructorColon: boolean;
+    vcFormatSpaceRemoveBeforeSemicolon: boolean;
+    vcFormatSpaceInsertAfterSemicolon: boolean;
+    vcFormatSpaceRemoveAroundUnaryOperator: boolean;
+    vcFormatSpaceBeforeFunctionOpenParenthesis: string;
+    vcFormatSpaceAroundBinaryOperator: string;
+    vcFormatSpaceAroundAssignmentOperator: string;
+    vcFormatSpacePointerReferenceAlignment: string;
+    vcFormatSpaceAroundTernaryOperator: string;
+    vcFormatWrapPreserveBlocks: string;
+    doxygenGenerateOnType: boolean;
+    doxygenGeneratedStyle: string;
+    doxygenSectionTags: string[];
+    filesExclude: Excludes;
+    filesAutoSaveAfterDelay: boolean;
+    filesEncoding: string;
     searchExclude: Excludes | undefined;
```

Comment:
It seems to me that `| undefined` should be removed from the searchExclude type.